src/main/java/ui/components/PanelMenuBar.java

```diff
@@ -186,6 +189,8 @@ private void augmentRenameableTextField(){
         menuBarNameArea.getChildren().addAll(renameableTextField);
         menuBarNameArea.setMinWidth(NAME_AREA_EDIT_WIDTH);
         menuBarNameArea.setMaxWidth(NAME_AREA_EDIT_WIDTH);
+        Tooltip.install(menuBarConfirmButton, new Tooltip("Confirm"));
+        Tooltip.install(menuBarUndoButton, new Tooltip("Undo"));
```

Comment:
Try changing the tip to this: "Abandon this name change and go back to the previous name". Post a screenshot on how it looks (I want to see if this tool tip is too long to display nicely).